Resources/docs/index.rst

```diff
@@ -387,3 +408,107 @@ This configuration will allow synchronizing with following commands:
     app/console ongr:sync:execute oxid.product
 
 ..
+
+Example multi-shops
+-------------------
+
+One of the ways to setup a multi-shop is by creating different environments_ for each shop.
+
+.. _environments: http://symfony.com/doc/current/cookbook/configuration/environments.html
+
+Settings for english OXID shop version "en", to be available on shopdomain.com/en:
+
+.. code-block:: yaml
+
+    parameters:
+        es_oxid_index_name: ongr_oxid_en
+..
+
+.. code-block:: yaml
+
+    ongr_elasticsearch:
+        connections:
+            oxid:
+                index_name: %es_oxid_index_name%
+..
+
+.. code-block:: yaml
+
+    ongr_oxid:
+        database_mapping:
+            oxid:
+                tags:
+                    @shop_tag: '_1'
+                    @lang_tag: '_1'
+                shop_id: 0
+                lang_id: 1
+..
+
+
+.. note:: Language tag and id should be taken from OXID.
```

Comment:
Expand upon this, how does OXID tag languages and shops and why are these tags necessary.